crates/compiler/test_gen/src/gen_tags.rs

```diff
@@ -907,7 +907,9 @@ fn alignment_in_multi_tag_pattern_match() {
                         { bool, int }
 
                     Empty ->
-                        { bool: Bool.false, int: 0 }
+                        # dev backend codegen bug means we cannot use this inline
+                        false = Bool.false
+                        { bool: false, int: 0 }
```

Comment:
the problem is that we have an assignment `Bool.false = Call Bool.false`, so `Bool.false` is used as a function and as a local symbol. That is an IR problem that we need to fix in mono.